docs/custom_palettes.rst

```diff
@@ -410,24 +626,28 @@ File Management
 4. **Backup**: Maintain copies of working themes
 
 Important Notes
----------------
+----------------
 
 Limitations and Considerations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**Cannot use with load_stylesheet()**: Custom palettes cannot be used with ``qdarkstyle.load_stylesheet()``. This function only works with the built-in DarkPalette and LightPalette. The function explicitly validates palette IDs and will exit with an error if you pass a custom palette with any ID other than 'dark' or 'light'.
+**Cannot use with load_stylesheet()**: Custom palettes with standard 'dark' and 'light' IDs currently cannot be used with ``qdarkstyle.load_stylesheet()``. This function only works with the built-in DarkPalette and LightPalette classes. However, by using standard 'dark' and 'light' IDs, custom themes follow the same architecture and could potentially be supported in future versions.
 
 **Resource Management**: Generated themes include resource files that must be properly imported if using Method 2.
 
 **Path Management**: Ensure proper path handling when loading QSS files in different environments.
 
 **Qt Binding Compatibility**: Generated resource files work with all Qt bindings (PyQt5, PyQt6, PySide2, PySide6).
 
+**Theme Architecture**: Custom themes follow the same dark/light pattern as built-in themes, providing consistency and potential future compatibility.
+
 Technical Details
 ~~~~~~~~~~~~~~~~~
 
 **Why load_stylesheet() doesn't support custom palettes**: The function validates palette IDs and only accepts 'dark' or 'light'. This is because the function needs to import the correct resource modules (darkstyle_rc or lightstyle_rc) that contain the compiled QSS and icon resources. Custom palettes don't have these pre-compiled resources available in the package, so they must be generated separately and loaded directly.
 
+**Future Compatibility**: By using standard 'dark' and 'light' IDs, custom themes follow the same architecture as built-in themes. This design choice makes it easier to potentially integrate custom themes with the main API in future versions while maintaining backward compatibility.
```

Comment:
This probably will depend on plans for the future of the package, maybe you discussed this with @ccordoba12 ?